data-table/table.js

```diff
@@ -91,6 +91,41 @@ function deleteColumn(numberOfColumns) {
      */
 }
 
+/**
+ * This function takes the id of the list container for the candidates names
+ * It then adds to the list another space for another candidate at the bottom of the list
+ * @param listContainer id of the list container
+ */
+function addCandidate(listContainer) {
```

Comment:
```suggestion
function addRow(wrapperDivId) {
```

1. Changes listContainer to wrapperDivId for consistency with how the variable is named elsewhere
2. DataTables should be entirely agnostic to rcvis - use "Columns" and "Rows" instead of "Rounds" and "Candidates"